_applications/var.md

```diff
@@ -29,97 +29,83 @@ sophisticated manner.
+experimentation, with a single Spark executor embedded within the application. Running
+a notebook on OpenShift provides a lot of advantages even if we aren't using scale-out
+compute:
 
-## Prerequisites
+1. if we have a project accessible from a public cloud, we can work on it from
+anywhere, with any device;
+2. we can easily share our work with colleagues; and
+3. our results will be easily reproducible, since repeatable deployments mean they
+won't depend on any details of our environment.
 
-The easiest way to get a Spark cluster going in your OpenShift project is to
-use [`oshinko-deploy.sh`][odsh]. This provides a streamlined installation for
-the Oshinko cluster manager into your project. Oshinko is a service to manage
-Spark clusters for OpenShift projects.
+## Install and run the notebook
 
-[odsh]: https://github.com/radanalyticsio/oshinko-webui/blob/master/tools/oshinko-deploy.sh
+Installing our notebook is very simple:  we just need to create an OpenShift project and install and run the notebook application.  Make sure you're logged in to OpenShift and have selected the right project, and then execute the following two commands:
 
-We'll need to use a special Spark worker image for this application. This
-worker image has a small amount of historical stock return data stored in its
-filesystem. For a real application, you'd want to get data from persistent
-storage, from a database, or from an object store service. For a demo, though,
-it's much easier to package up a small amount of data where each worker can get
-it.
+`oc new-app radanalyticsio/workshop-notebook -e JUPYTER_NOTEBOOK_PASSWORD=developer`
 
-In order to get this special Spark worker image, we'll use the `-s` flag to
-`oshinko-deploy.sh`:
+`oc expose svc/workshop-notebook`
 
-`oshinko-deploy.sh -s willb/var-spark-worker`
+Check the OpenShift web console.  Once the notebook application is running, click on its route.  Jupyter will ask you to log in; use `developer` for a password (or the password you specified if you used a different one when you ran `oc new-app`).  You'll be presented with a list of notebooks that are installed in the workshop-notebook image.  The one we're interested in is `var.ipynb`, so click on it.  (There are some other files in this image:  `pyspark.ipynb` is an introduction to Apache Spark, `ml-basics.ipynb` introduces some basic machine learning techniques, and `var-demo.ipynb` is similar to `var.ipynb`, but with less explanatory text, to make it more suitable for a live demo.)
 
-## Setting up a cluster
+Interacting with the Jupyter notebook is very simple. Select a cell with Python
+code and then execute it, either by pressing the "run cell" button in the toolbar, or by pressing &#8679;&#8629;.  You can edit the code in the notebook cells and re-run them, and results from cells you've already run will be available to new cells. The notebook interface provides a great way to experiment with new techniques. Try it out!
```

Comment:
minor nit, as much as i love `&#8679;&#8629;` i think we should add `(shift + enter)` after the icons